### DISCUSSION TITLE
Consider adding Clarabel solver to the benchmarks

### DISCUSSION THREAD

**Discussion by nrontsis** [qpsolvers/qpbenchmark#45] [Ideas]
[Clarabel](https://docs.juliahub.com/Clarabel/g676L/0.1.2/) is a new solver that was shown to be competitive in some QP problems of mine.

See related discussion [here](https://discourse.julialang.org/t/ann-clarabel-jl-new-convex-optimization-solver/83915) and related talk [here](https://www.youtube.com/watch?v=NSPhIbXk9n4).

**Comment by stephane-caron** [comment]
Thanks for pointing out this solver :+1:  Tracking its integration in https://github.com/stephane-caron/qpsolvers/issues/173 then https://github.com/stephane-caron/qpsolvers_benchmark/issues/47.

**Reply by nrontsis** [reply]
Thanks! 🙌
